mono/tests/tailcall-virt.cs

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
This file is not used anywhere and thus useless. Please remove.

Reply:
It is the source for tailcall-virt.il.

Reply:
Ideally such assert(f) has a name instead of number..maybe we have that available..